### ISSUE TITLE
Bot fails to pick up older major versions

### ISSUE THREAD

**Issue by atniomn** [regro/cf-scripts#2365]
https://github.com/conda-forge/conan-feedstock/issues/206#issue-2209645317

The conan feedstock uses this bot. After Conan 2.x release, bot fails to pick up any new 1.x releases, in particular it has failed to pick up 1.60.0, 1.61.0, 1.62.0, 1.63.0, etc... There is still a substantial community of people using Conan 1.x (often in parallel with Conan 2.x), so it would nice to have those versions.

Furthermore, I am not sure Conan is not the only package that does this. Bot seems like it should be able to pick up new minor relases of older major versions, in general.

**Comment by beckermr** [comment]
This is a known limitation. It will likely be difficult to fix given how the bot is currently structured and how release branches are used in conda-forge.
